From a2ab98517d7a25ffcf81c68439eb95276d016607 Mon Sep 17 00:00:00 2001
From: Thomas Gleixner <tglx@linutronix.de>
Date: Tue, 13 Aug 2013 16:08:41 -0400
Subject: [PATCH 117/318] panic: skip get_random_bytes for RT_FULL in
 init_oops_id

---
 kernel/panic.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/kernel/panic.c b/kernel/panic.c
index e1b2822..5dc4381 100644
--- a/kernel/panic.c
+++ b/kernel/panic.c
@@ -371,9 +371,11 @@ static u64 oops_id;
 
 static int init_oops_id(void)
 {
+#ifndef CONFIG_PREEMPT_RT_FULL
 	if (!oops_id)
 		get_random_bytes(&oops_id, sizeof(oops_id));
 	else
+#endif
 		oops_id++;
 
 	return 0;
-- 
1.8.4.rc3

