From 0d2e2a6d6e0f5d0acdd3b96d895342d1a71a00e4 Mon Sep 17 00:00:00 2001
From: "Steven Rostedt (Red Hat)" <rostedt@goodmis.org>
Date: Wed, 11 Sep 2013 19:44:46 -0400
Subject: [PATCH 313/313] Linux 3.8.13-rt16 REBASE

---
 localversion-rt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 08b3e75..1199eba 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt14
+-rt16
-- 
1.7.9.5

